src/ResourceManager/KeyVault/Commands.KeyVault/help/Set-AzureKeyVaultManagedStorageSasDefinition.md

```diff
@@ -141,28 +26,26 @@ Storage Account.
 
 ## EXAMPLES
 
-### Example 1 : Set an ad hoc service Blob sas definition
-```
-PS C:\> Set-AzureKeyVaultManagedStorageSasDefinition -Blob 'blob1' -Container 'container1' -VaultName 'vault1' -AccountName 'account1' -Name 'sas1' -ValidityPeriod ([System.Timespan]::FromDays(30)) -Permission Read,Add -SharedAccessHeader CacheControl,ContentDisposition -Protocol HttpsOnly -IPAddressOrRange '168.1.5.60-168.1.5.70'
-```
-
-Sets an ad hoc service blob sas definition 'sas1' with key vault managed storage account 'account1' in vault 'vault1'.
-
-### Example 2 : Set an ad hoc account sas definition
-```
-PS C:\> Set-AzureKeyVaultManagedStorageSasDefinition -Service Blob,File -ResourceType Container,Service -VaultName 'vault1' -AccountName 'account1' -Name 'sas1' -Protocol HttpsOrHttp -IPAddressOrRange '168.1.5.60' -ValidityPeriod ([System.Timespan]::FromDays(30)) -Permission Read,Add
-```
-
-Sets an ad hoc blob sas definition 'sas1' with key vault managed storage account 'account1' in vault 'vault1'.
-
-### Example 3 : Set a sas definition using a hashtable
-```
-PS C:\> $parameters = @{"sasType"="blob";"signedVersion"="2016-05-31";"signedProtocols"="https";"signedIp"="168.1.5.60-168.1.5.70";"validityPeriod"="P30D";"signedPermissions"="ra";"blobName"="blob1";"containerName"="container1";"rscd"="";"rscc"=""}
-PS C:\> Set-AzureKeyVaultManagedStorageSasDefinition -VaultName vault1 -AccountName account1 -Name sas1 -Parameter $parameters
-```
-
-Sets an ad hoc blob sas definition 'sas1' with key vault managed storage account 'account1' in
-vault 'vault1' using a hashtable.
+### Example 1 : Set an account-type SAS definition, and obtain a current SAS token based on it
+```
+PS C:\> $sa=Get-AzureRmStorageAccount -Name mysa -ResourceGroupName myrg
+PS C:\> $kv=Get-AzureRmKeyVault -VaultName mykv
+PS C:\> Add-AzureKeyVaultManagedStorageAccount -VaultName $kv.VaultName -AccountName $sa.StorageAccountName -AccountResourceId $sa.Id -ActiveKeyName key1 -RegenerationPeriod 180
+PS C:\> $sctx=New-AzureStorageContext -StorageAccountName $sa.StorageAccountName -Protocol Https -StorageAccountKey Key1
```

Comment:
Does running this require access to the actual storage account key?  Can users provide a raw "dummy" key such as 00000000?